specs/reporters/crash.cpp

```diff
@@ -10,7 +10,7 @@ go_bandit([]() {
 
     before_each([&]() {
       stm.str(std::string());
-      reporter.reset(new reporter::crash(stm, formatter));
+      reporter.reset(new reporter::crash(stm, formatter, false));
```

Comment:
As mentioned in :abc:, this should not be necessary because the reporter does not support timing reports.

Reply:
Fixed here: 043991d